Sources/tinys3/model/Multipart/MultipartUploadFile.swift

```diff
@@ -39,7 +27,7 @@ actor MultipartUploadFile {
             try handle.seek(toOffset: UInt64(range.lowerBound))
 
             if #available(macOS 10.15.4, *) {
-                guard let data = try handle.read(upToCount: range.count + 1) else {
+                guard let data = try handle.read(upToCount: range.count) else {
```

Comment:
The second seed of the off-by-one error: the interpretation of each `Range` as a `ClosedRange` during the computation of `var parts` was compensated here by the `range.count + 1` that effectively made the upper bound of the `Range` object be included in the bytes being read, so in the end no bytes was missed due to this over-correction, but each part being read was 1 extra byte compared to the expected `partSize` number.